src/nvim/highlight_group.h

```diff
@@ -12,7 +12,7 @@ typedef struct {
   char *name;
   RgbValue color;
 } color_name_table_T;
-extern color_name_table_T color_name_table[700];
+extern color_name_table_T color_name_table[709];
```

Comment:
Does 708 work now?

Reply:
That is super strange. Putting definition of new colors *before* their `***Green` counterpart in the array (to be alphabetical) is what made 708 work. At least for those new colors.

For more context:
- Have `extern color_name_table_T color_name_table[708];`.
- With the following order executing `:hi aaa guifg=NvimLightGray1 | hi aaa` shows `aaa xxx cleared`:
```
  { "NvimLightGreen", RGB_(0xb3, 0xf6, 0xc0) },
  { "NvimLightGray1", RGB_(0xee, 0xf1, 0xf8) },
  { "NvimLightGray2", RGB_(0xe0, 0xe2, 0xea) },
  { "NvimLightGray3", RGB_(0xc4, 0xc6, 0xcd) },
  { "NvimLightGray4", RGB_(0x9b, 0x9e, 0xa4) },
  { "NvimLightGrey1", RGB_(0xee, 0xf1, 0xf8) },
  { "NvimLightGrey2", RGB_(0xe0, 0xe2, 0xea) },
  { "NvimLightGrey3", RGB_(0xc4, 0xc6, 0xcd) },
  { "NvimLightGrey4", RGB_(0x9b, 0x9e, 0xa4) },
```
- But with the following ordering the previous command works:
```
  { "NvimLightGray1", RGB_(0xee, 0xf1, 0xf8) },
  { "NvimLightGray2", RGB_(0xe0, 0xe2, 0xea) },
  { "NvimLightGray3", RGB_(0xc4, 0xc6, 0xcd) },
  { "NvimLightGray4", RGB_(0x9b, 0x9e, 0xa4) },
  { "NvimLightGreen", RGB_(0xb3, 0xf6, 0xc0) },
  { "NvimLightGrey1", RGB_(0xee, 0xf1, 0xf8) },
  { "NvimLightGrey2", RGB_(0xe0, 0xe2, 0xea) },
  { "NvimLightGrey3", RGB_(0xc4, 0xc6, 0xcd) },
  { "NvimLightGrey4", RGB_(0x9b, 0x9e, 0xa4) },
```

So I guess alphabetical order is a **must** here?

Reply:
Yes, because of binary search.

Reply:
Do you still prefer that "Goldenrod" is spelled consistently (with Vim) as a part of this PR?

Reply:
Yes. It's single-line change and there is no other PR to include it in.

Reply:
> Yes. It's single-line change and there is no other PR to include it in.

Not quite. Vim consistently has "Goldenrod" spelled with the lower "r" (whenever "G" is an upper). But no problem.

Reply:
Replaced all "GoldenRod"s with "Goldenrod"s. Also checked that they can be assigned in highlight group (with 708 array size).